src/components/icon-button/icon-button.scss

```
@@ -9,10 +9,12 @@
 }
 
 /**
- * @prop --icon-background-color: Background color when attribute `elevated` is set to `true`.
+ * @prop --icon-button-background-color: Background color when attribute `elevated` is set to `true`.
  */
 
 :host {
+    --specified-background-color: var(--icon-button-background-color),
+        (var(--icon-background-color));
```

Does this work? Shouldn't it be:

```suggestion
    --specified-background-color: var(--icon-button-background-color,
        var(--icon-background-color));
```

---

When did we make this PR? Totally forgot about it. Anyhow, today, we change this `var(--lime-elevated-surface-background-color)` in any component that needs another color for their elevated surface.